packages/theme/src/fonts/index.ts

```diff
@@ -5,16 +5,32 @@
  */
 export const primary = {
   family: 'Lato, sans-serif',
-  variant: {
-    p1: '2rem',
-    p2: '1.8rem',
-    p3: '1.6rem',
-    p4: '1.4rem',
-    caption1: '1.2rem',
-    button1: '2rem',
-    button2: '1.8rem',
-    button3: '1.6rem',
-    button4: '1.4rem',
+  p1: {
+    fontSize: '2rem',
+  },
+  p2: {
+    fontSize: '1.8rem',
+  },
+  p3: {
+    fontSize: '1.6rem',
+  },
+  p4: {
+    fontSize: '1.4rem',
+  },
+  caption1: {
+    fontSize: '1.2rem',
+  },
+  button1: {
+    fontSize: '2rem',
+  },
+  button2: {
+    fontSize: '1.8rem',
+  },
+  button3: {
+    fontSize: '1.6rem',
+  },
+  button4: {
+    fontSize: '1.4rem',
   },
```

Comment:
What do you think about adding another layer of nesting?

```ts
{
  button: {
    small: ...,
    medium: ...,
    ...
  }
}
```

We can do this for `p` and `caption` as well

Reply:
Updated